modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -1511,6 +1512,18 @@
 		end
 	end
 
+	function m.allModulesPublic(cfg)
+		if _ACTION >= "vs2019" then
+			if cfg.allmodulespublic ~= nil then
+				if cfg.allmodulespublic then
+					m.element("AllProjectBMIsArePublic", nil, "true")
```

Comment:
For future reference, you can use `iif(<cond>, "true", "false")` instead of duplicating the line.

Reply:
Which line?

Reply:
To follow up, that would take the place of the if-else block. It'd be:

```lua
m.element("AllProjectBMIsArePublic", nil, iif(cfg.allmodulespublic, "true", "false"))
```